src/app/pages/home-page/vendor-item-page/vendor-item-page.component.html

```diff
@@ -0,0 +1,40 @@
+<div *ngIf="vendor$ | async as vendor" class="vendor">
+  <div class="vendor__card">
+    <img
+      src="/assets/images/vendor_logo.jpg"
+      alt="vendor logo"
+      class="vendor__image"
+    />
+    <div>
+      <div class="vendor__name">{{ vendor.name }}</div>
+      <div class="vendor__title">{{ vendor.title }}</div>
+    </div>
+    <a href="https:{{ vendor.website }}" class="vendor__link link-reset">
+      <mat-icon>public</mat-icon>Website
+    </a>
+  </div>
+  <mat-tab-group mat-stretch-tabs class="tab">
+    <mat-tab label="offers">there will be offers</mat-tab>
+    <mat-tab label="description">{{ vendor.description }}</mat-tab>
+    <mat-tab label="addresses">
+      <div
+        *ngFor="let office of vendor.offices"
+        class="address"
```

Comment:
classes addresses and address so similar.

Reply:
but I have only "address", "addresses" it's a label

Reply:
The problem is "addresses" and "address" have almost the same spelling, so programmers can make a mistake during write code. I recommend you change "addresses" to "group-of-addresses"

Reply:
I can't change the label as this is the title of the tab. and if we rename it to "group-of-addresses", it will not change problem you wrote about. I do not see this as a serious problem